src/screenshot/screenshot.c

```diff
@@ -57,6 +57,7 @@ static int method_screenshot(sd_bus_message *msg, void *data,
 	// TODO: choose a better path
 	const char path[] = "/tmp/out.png";
 	if (!exec_screenshooter(path)) {
+        xdpw_request_destroy(req);
```

Comment:
Style: we use tabs for indentation

Reply:
oops - my editor wasn't set to tabs, let me fix it